.github/workflows/release.yml

```
@@ -52,6 +52,16 @@ jobs:
           cd ./publish/linux-arm64 && tar -czf ../../dotnet-api-diff-linux-arm64.tar.gz . && cd ../..
           cd ./publish/osx-arm64 && tar -czf ../../dotnet-api-diff-osx-arm64.tar.gz . && cd ../..
 
+      - name: Calculate checksums
+        id: checksums
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)